src/Notifications/AbstractNotification.php

```diff
@@ -63,4 +72,92 @@ abstract class AbstractNotification
 
         return config($config_key, []);
     }
+
+    /**
+     * Return the class of a specific provider implementation.
+     *
+     * @param string $provider
+     *
+     * @return string
+     * @throws \Herpaderpaldent\Seat\SeatNotifications\Exceptions\UnknownDriverException
+     * @example
+     * ```
+     *   ProviderANotificationImplementationClass
+     * ```
+     */
+    final public static function getDiverImplementation(string $provider): string
+    {
+        try {
+            // build the configuration key related to the notification
+            $config_key = sprintf('services.seat-notification.%s.%s', get_called_class(), $provider);
+
+            return config($config_key, []);
+        } catch (\Throwable $exception) {
+
+            // throw exception if implementation is not found.
+            throw new UnknownDriverException($provider);
+        }
+    }
+
+    /**
+     * @var array
+     */
+    protected $tags = [];
+
+    /**
+     * @var array
+     */
+    protected $colors = [
+        'danger' => ['hex' => '#dd4b39', 'dec' => '14502713'],
```

Comment:
That should be moved as a constant into the implementation.

Abstract should only contain dataset definitions. 
The style is for implementation.